src/Microsoft.AspNetCore.Server.Testing/Deployers/NanoDeployer/RemotePSSessionHelper.ps1

```diff
@@ -0,0 +1,44 @@
+﻿[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$serverName,
+
+	[Parameter(Mandatory=$true)]
+	[string]$accountName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$accountPassword,
+
+    [Parameter(Mandatory=$true)]
+    [string]$executablePath,
+
+    [Parameter(Mandatory=$true)]
+    [string]$serverType,
+	
+	[Parameter(Mandatory=$true)]
+	[string]$serverAction
+)
+
+Write-Host "`nExecuting deployment helper script on machine '$serverName'"
+Write-Host "`nSarting a powershell session to machine '$serverName'"
+
+$securePassword = ConvertTo-SecureString $accountPassword -AsPlainText -Force
+$credentials= New-Object System.Management.Automation.PSCredential ($accountName, $securePassword)
+$psSession = New-PSSession -ComputerName $serverName -credential $credentials
+
+if ($serverAction -eq "StartServer")
+{
+	Write-Host "Starting the application on machine '$serverName'"
+	$startServerScriptPath = "$PSScriptRoot\StartServer.ps1"
+	Invoke-Command -Session $psSession -FilePath $startServerScriptPath -ArgumentList $executablePath, $serverType, $serverName
```

Comment:
You'll have no idea if this script succeeded. Look at the code here to see how to get the exit code https://github.com/aspnet/MusicStore/blob/dev/test/RemoteTest.ps1#L25-L34